package.json

```diff
@@ -29,6 +29,7 @@
     "ioredis": "^4.28.2",
     "jest": "^27.5.1",
     "npm-run-all": "^4.1.5",
+    "p-queue": "^7.2.0",
```

Comment:
This should be a `devDependency` in the `packages/wrangler/package.json` file.

The way to add deps to packages is from the root of the monorepo via

```sh
npm i -w wrangler -D p-queue
```

Reply:
Should it? It's being used to manage concurrency on the Pages file uploads... It's not a dev dependency at all, I don't think?